server/entity/arrow.go

```diff
@@ -31,24 +32,27 @@ type Arrow struct {
 	owner world.Entity
 	tip   potion.Potion
 
+	fireTicks  int64
+	punchLevel int
+
 	disallowPickup, obtainArrowOnPickup bool
 
 	c *ProjectileComputer
 }
 
 // NewArrow creates a new Arrow and returns it. It is equivalent to calling NewTippedArrow with `potion.Potion{}` as
 // tip.
-func NewArrow(pos mgl64.Vec3, yaw, pitch float64, owner world.Entity) *Arrow {
-	return NewTippedArrow(pos, yaw, pitch, owner, potion.Potion{})
+func NewArrow(pos mgl64.Vec3, yaw, pitch, damage float64, owner world.Entity) *Arrow {
+	return NewTippedArrow(pos, yaw, pitch, damage, owner, potion.Potion{})
 }
 
 // NewTippedArrow creates a new Arrow with a potion effect added to an entity when hit. The Arrow's base damage may be
 // changed by calling Arrow.SetBaseDamage. It is 2.0 by default.
```

Comment:
This documentation here isn't right anymore. Might want to correct this too.

Reply:
Correct.d